### DISCUSSION TITLE
Отправка запроса на настройку роуминга через COM

### DISCUSSION THREAD

**Discussion by W1tchcr4ft3r** [diadoc/diadocsdk-1c-docs#1088] [Ideas]
Здравствуйте.
Подскажите, пожалуйста, не появилось ли возможности отправки заявки на роуминг через COM объект, без необходимости открытия веб-интерфейса? Или этого не планируется в пользу развития автороуминга ?

**Comment by GilimkhanovDenis** [comment]
Добрый день

Пока данной функциональности не планировалось
